scripts/calculate_avg_tat.sql

```diff
@@ -29,4 +29,15 @@ join family_sample as f_s on f_s.`sample_id` = s.id
 join family as f on f.id = f_s.`family_id`
 join analysis as a on a.`family_id` = f.id
 where s.delivered_at is not null and a.uploaded_at is not null
-and DATEDIFF(s.delivered_at, a.uploaded_at) >= 0 AND DATEDIFF(s.delivered_at, a.uploaded_at) <= 31;
\ No newline at end of file
+and DATEDIFF(s.delivered_at, a.uploaded_at) >= 0 AND DATEDIFF(s.delivered_at, a.uploaded_at) <= 31;
+
+-- O_A external
+select avg(DATEDIFF(a.completed_at, f.ordered_at))
```

Comment:
Is this query here just to test the query out?

Reply:
The code has some hard-coded values that has been calculated via the script. The script can be used if you want to verify the hard-coded values or if you want to calculate and update the values.